lib/constants/limits.ts

```diff
@@ -1,9 +1,15 @@
-/** Order size limits denominated in satoshis. */
-const limits: { [currency: string]: number } = {
+/** Mainnet order size limits denominated in satoshis. */
+export const limits: { [currency: string]: number }  = {
   BTC: 100000,
   LTC: 15000000,
   WETH: 5000000,
   DAI: 1000000000,
 };
 
-export default limits;
+/** The maximum network channel sizes for each currency denominated in satoshis. */
+export const maxLimits: { [currency: string]: number } = {
+  BTC: 16777216, // 4294967 sat payment limit lifted as of 2019
```

Comment:
I actually didn't know that, great!

Reply:
Yep, I found that out while working on this PR. https://twitter.com/Stadicus3000/status/1088391443067080704

Reply:
Reading through the thread, it seems for direct channels only, routed payments (which actually use HTLCs) are still subject to the 0.04 BTC limit. Anyhow, it's ok to lift it to 0.16 for our purpose since we expect to have a good amount of direct channels.

Reply:
Ahh gotcha, but yeah it makes sense for an upper bound which is what we're aiming for here.